packages/mui-material-next/src/CircularProgress/CircularProgress.tsx

```diff
@@ -188,7 +230,7 @@ const CircularProgress = React.forwardRef(function CircularProgress<
   return (
     <CircularProgressRoot
       className={clsx(classes.root, className)}
-      style={{ width: size, height: size, ...rootStyle, ...style }}
```

Comment:
We should bring back `...style` here and destructure it from props. With these changes, if the user provides a `style` prop, `rootStyle` would be overridden.

Reply:
Fixed it